vshard/init.lua

```diff
@@ -1,4 +1,5 @@
 return {
+    _VERSION = require('vshard.consts').VERSION,
```

Comment:
Well, maybe it's better to require `consts` a single time

Reply:
Done